index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
- [ ] You did great work 😃, however, you need to fix some parts. For good syntax, Kindly make sure you follow this structure:

`````
<body>
    <main>
        <header>
               <a href="#">LOGO</a>
               <nav>
                    --- hamburger menu ---
               </nav>
        </header>
        <section>
               --- the headline section (right after the header) ----
        </section>
    </main>
</body>
`````


- [ ]  Kindly make sure your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&scaling=min-zoom&page-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃,

> Your `background image`
> Your h2 : `color: #172B4D;`
> Your p: `color: #344563;` 


> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-05-04 122601](https://user-images.githubusercontent.com/57408419/236179015-1d96c758-835d-4423-964c-f76b5a69d3df.png)| |![Screenshot 2023-05-04 122734](https://user-images.githubusercontent.com/57408419/236179118-edf18554-5084-45c2-b462-2615a8c98a5c.png)| |